src/main/java/com/testsigma/sdk/RestAction.java

```diff
@@ -0,0 +1,4 @@
+package com.testsigma.sdk;
+
+public abstract class RestAction extends DriverAction{
```

Comment:
@SunilGembali Should this be extended from DriverAction? RestApi will not have any driver. So please check and change it, we shouldn't be exposing any driver for RestAPI ApplicationType